README.md

```diff
@@ -93,7 +93,7 @@ x86, x86-64, LoongArch64, 32-bit & 64-bit Little Endian MIPS, 32-bit PowerPC (Bi
 support WebAssembly.
 For more information, see [the supported `ring` target platforms][ring-target-platforms].
 
-By providing a custom implementation of the [`crate::crypto::CryptoProvider`] trait, you
+By providing a custom implementation of the [`crate::crypto::CryptoProvider`] struct, you
```

Comment:
"custom implementation of the (...) struct" is odd. "customised value of the (...) struct" maybe?

Reply:
Good call, reworded.

Reply:
Sorry, this is very nitpicky, but this still feels like it's about the fields of the struct (which have like a name and a type), not the values going in the fields. Maybe "By providing a customized instance [or value] of the [..] type"?

Reply:
Switched to that language.